### PR TITLE
PTL-1079 Notify improvemnts

### PR DESCRIPTION
**Pull request by wjhendry**
Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?
No. Project structure about to change, so I'm not going to update Previous at this stage

Have you checked all new or changed links?
Yes

Is there anything else you would like us to know?
To be checked by @tgosling-genesis 

For reference: 

  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 

**This week's exciting excerpts from the style guide**

- We write in UK English, with Oxford English Dictionary (OED) spellings, grammar and vocabulary.  

Use the present tense wherever possible. Only use another tense where it is strictly necessary.

- Present tense (preferred whenever possible): *The script creates a new folder.*
- Future tense (avoid whenever possible): *The script will create a new folder.*

### CODE REVIEW

File: docs/03_server/10_integration/06_notify/03_symphony.md [hunk 1]
```diff
@@ -14,51 +14,77 @@ tags:
...
-### Adding the Symphony module endpoints to GENESIS_NOTIFY
-In order to add the Symphony endpoints to your GENESIS_NOTIFY process, add the following package in your process definition: 
-```
-global.genesis.symphony.gateway.endpoints
-```
-and the following jar to the GENESIS_NOTIFY_PROCESS classpath
+## Running locally
+To run locally, you need to copy your **genesis-notify-processes.xml** file. If you are using the IntelliJ plugin, you can find this in the folder **.genesis-home/genesis-notify/cfg/**.
+
+Paste this file into the **site-specific** folder at **server/jvm/alpha-site-specific/src/main/resources/cfg/**.
```
Comment (tgosling-genesis) [inline docs/03_server/10_integration/06_notify/03_symphony.md:50]: Should we use alpha specifically outside of the training

**Review by tgosling-genesis — APPROVED** [review]

File: docs/03_server/10_integration/06_notify/03_symphony.md [hunk 2]
```diff
@@ -14,51 +14,77 @@ tags:
...
+
+```xml
+<classpath>genesis-symphony-manager*</classpath>
 ```
 
-### Adding Symphony extensions for notify GPAL scripts
-To configure symphony gateways, you need to import extensions for the Notify GPAL configuration. Add the following dependency to your application script-config module:
+Next, you need to import the extensions for the Notify GPAL configuration. This enables intellisense to work for your Notify scripts. 
+
+To do this, add the following dependency to the **build.gradle.kts** file in your application's **server/jvm/gradle** folder:
```
[the remark this answers is not in the document]
Comment (wjhendry) [reply docs/03_server/10_integration/06_notify/03_symphony.md:69]: noted

[the remark this answers is not in the document]
Comment (GuiWar) [reply docs/03_server/10_integration/06_notify/03_symphony.md:69]: server/jvm/{app-name}-scrip-config/

File: docs/03_server/10_integration/06_notify/03_symphony.md (hunk 1 above)
Comment (wjhendry) [reply docs/03_server/10_integration/06_notify/03_symphony.md:50]: No, I should have changed that and I shall shortly

**Review by GuiWar — APPROVED** [review]

**Pull request merged**